map.js

```diff
@@ -201,12 +202,13 @@ SH: 'SHP',
 SI: 'EUR',
 SJ: 'NOK',
 SK: 'EUR',
-SL: 'SLL',
+SL: 'SLE',
 SM: 'EUR',
 SN: 'XOF',
 SO: 'SOS',
 SR: 'SRD',
-ST: 'STD',
+SS: 'SSP',
+ST: 'STN',
```

Comment:
[Sao Tome and Principe](https://en.wikipedia.org/wiki/S%C3%A3o_Tom%C3%A9_and_Pr%C3%ADncipe) uses 
[São Tomé and Príncipe dobra](https://en.wikipedia.org/wiki/S%C3%A3o_Tom%C3%A9_and_Pr%C3%ADncipe_dobra)